client/src/main/java/com/tigrisdata/db/client/StandardTigrisAsyncDatabase.java

```diff
@@ -171,6 +177,33 @@ public CompletableFuture<DatabaseDescription> describe() throws TigrisException
         DESCRIBE_DB_FAILED);
   }
 
+  @Override
+  public void stream(TigrisAsyncStreamer streamer) {
+    Api.StreamRequest streamRequest = Api.StreamRequest.newBuilder().setDb(db).build();
+    stub.stream(
+        streamRequest,
+        new StreamObserver<Api.StreamResponse>() {
+          @Override
+          public void onNext(Api.StreamResponse streamResponse) {
+            try {
+              streamer.onNext(StreamEvent.from(streamResponse.getEvent(), objectMapper));
+            } catch (IOException e) {
+              streamer.onError(new TigrisException("Failed to convert event data to JSON", e));
```

Comment:
Please move this error message to Messages constant holder class.

Reply:
k